### PR TITLE
Fixes #25342: Changes for modules packaging

### PR DESCRIPTION
**Pull request by amousset**
https://issues.rudder.io/issues/25342

### CODE REVIEW

**Comment by amousset** [comment]
PR updated with a new commit

File: rust.makefile [hunk 1]
```diff
@@ -12,13 +12,14 @@ version:
...
 	@echo "RUSTC_WRAPPER=$${RUSTC_WRAPPER}"
+	# sccache is used if present
 	sccache --show-stats || true
 
 # https://matklad.github.io/2021/09/04/fast-rust-builds.html#ci-workflow
 build: CARGO_INCREMENTAL=0
 build: version
 	cargo install --locked cargo-auditable@0.6.1
-	cargo auditable build --release --locked
+	cargo auditable build ${CARGO_FEATURES} --release --locked
```
Comment (peckpeck) [inline rust.makefile:22]: where does this variable come from ?

Comment (amousset) [reply rust.makefile:22]: this PR goes with https://github.com/Normation/rudder-packages/pull/2901

**Review by peckpeck — APPROVED** [review]

**Comment by amousset** [comment]
OK, squash merging this PR

**Pull request merged**